mssqlcli/mssql_cli.py

```diff
@@ -667,6 +685,16 @@ def has_change_db_cmd(self, query):
 
         return False, None
 
+    def has_security_statement_in_cmd(self, query):
+        """Determines if the statement contains a PASSWORD or SECRET keyword
+        """
+        try:
+            tokens = query.lower()
+            return 'password' in tokens or 'secret' in tokens
```

Comment:
Are these the only tokens we want to ignore? Is there any value in making a list of forbidden words and iterating over them to check if any tokens match that?

Reply:
ah yes, we can have a top level list that both MssqlCliFileHistory and this function can reference.